### PR TITLE
fix(suite): bump BTC fee flow

### LINKED ISSUE #9846: Discrepancy when bumping and finalizing bitcoin transaction
**Describe the bug**
There is discrepancy when bumping and finalizing bitcoin transaction.
There are 3 steps in modal but last thing you confirm on device is fee rate and modal still shows step 2 of 3.
And there is no hold to confirm or press both buttons to confirm on device at end of confirmation.
Same reults for model Safe 3 and model T

**Steps to reproduce:**
1. Send bitcoin transaction
2. Bump transaction
3. Compare Trezor Suite modal and device dialogues

**Info:**
 - Suite version: desktop 33.11.1 (a70b4d364b1233d4ed620473c04755f34a798acd)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/33.11.1 Chrome/116.0.5845.228 Electron/26.4.2 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: Trezor T2B1 2.6.3 regular (revision 2c7cc6e0255dee2339b445b5551eaffb88dbd1b4)
 - Device: Trezor T2T1 2.6.3 regular (revision 2c7cc6e0255dee2339b445b5551eaffb88dbd1b4)
 - Transport: BridgeTransport 2.0.32

**Screenshots:**
telegram-cloud-photo-size-4-5920023247263483221-y

https://github.com/trezor/trezor-suite/assets/31506317/ab04f07a-43e8-49c4-83a6-6aaedb21c95f

### PR DESCRIPTION
**Pull request by AdamSchinzel**
## Description

Fixes the amount of steps when bumping BTC fee.

## Related Issue

Resolve #9846 

## Screenshots:
C79CE975-0AAD-4496-B754-D0F842E1895A

### CODE REVIEW

File: packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx [hunk 1]
```diff
@@ -159,6 +162,10 @@ export const TransactionReviewOutputList = ({
         }
     }, [buttonRequestsCount, outputs.length]);
 
+    if (!device) return null;
+
+    const isUpdatedRbfBtcFlowFlow = getIsUpdatedRbfFlow(device);
```
Comment (komret) [inline packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx]: typo (FlowFlow)

File: packages/suite/src/utils/wallet/reviewTransactionUtils.ts [hunk 2]
```diff
@@ -21,6 +21,12 @@ export const getIsUpdatedSendFlow = (device: TrezorDevice) => {
     return versionUtils.isNewerOrEqual(firmwareVersion, '2.6.0');
 };
 
+export const getIsUpdatedRbfFlow = (device: TrezorDevice) => {
+    const firmwareVersion = getFirmwareVersion(device);
+
+    return versionUtils.isNewerOrEqual(firmwareVersion, '2.6.3');
+};
+
```
Comment (matejkriz) [inline packages/suite/src/utils/wallet/reviewTransactionUtils.ts]: There is no need for this new condition, because it's not a new change, but old bug.

Comment (AdamSchinzel) [reply packages/suite/src/utils/wallet/reviewTransactionUtils.ts]: 0d367da89c74a32712162743a3b9b0b3cc8944a1

File: packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx [hunk 3]
```diff
@@ -82,6 +82,7 @@ export interface TransactionReviewOutputListProps {
     outputs: ReviewOutput[];
     buttonRequestsCount: number;
     isRbfAction: boolean;
+    network?: Network;
```
Comment (matejkriz) [inline packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx]: leftover?

Comment (AdamSchinzel) [reply packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx]: 4a5ddbbe3ce81e4f460e46c0be13c4bec6062881

Comment (matejkriz) [reply packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx]: not complete, please check failing validation

Comment (AdamSchinzel) [reply packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx]: Sorry, now it should be fixed feb1f0345d4131b63d2041a745b5534fda51fde0

**Review by matejkriz — APPROVED** [review]
🚢

**Pull request merged**